dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -1440,6 +1462,8 @@ def _find_all_providers_affected(self, include_docs: bool) -> list[str] | AllPro
                     all_providers.add(provider)
         if self.needs_api_tests:
             all_providers.add("fab")
+        if self.needs_ol_tests:
+            all_providers.add(OPENLINEAGE_INTEGRATION)
```

Comment:
I don't think we should use "OPENLINEAGE_INTEGRATION" here. This is really constant for "integration" name, that coincides with "openlineage" provider name, but that's pure coincidence- there is for example "STATSD_INTEGRATION" that does not have provider. 

I think (see few lines above "fab" it's ok to use hardcoded provider names in this case - it's unlikely we will ever change them and it's a bit more explicit here what we are trying to pass - provider id, rather than integration name.

Reply:
Thank you for the suggestion. I have implemented the suggested changes.